inlong-dashboard/src/metas/sinks/defaults/Hive.ts

```diff
@@ -117,39 +117,33 @@ export default class HiveSink extends SinkInfo implements DataWithBackend, Rende
   password: string;
 
   @FieldDecorator({
-    type: 'input',
-    rules: [{ required: true }],
-    props: values => ({
-      disabled: [110, 130].includes(values?.status),
-      placeholder: 'jdbc:hive2://127.0.0.1:10000',
-    }),
-  })
-  @I18n('JDBC URL')
-  jdbcUrl: string;
-
-  @FieldDecorator({
-    type: 'input',
-    rules: [{ required: true }],
-    tooltip: i18n.t('meta.Sinks.DataPathHelp'),
-    props: values => ({
-      disabled: [110, 130].includes(values?.status),
-      placeholder: 'hdfs://127.0.0.1:9000/user/hive/warehouse/default',
-    }),
-  })
-  @I18n('meta.Sinks.Hive.DataPath')
-  dataPath: string;
-
-  @FieldDecorator({
-    type: 'input',
+    type: 'select',
     rules: [{ required: true }],
-    tooltip: i18n.t('meta.Sinks.Hive.ConfDirHelp'),
     props: values => ({
+      showSearch: true,
       disabled: [110, 130].includes(values?.status),
-      placeholder: '/usr/hive/conf',
+      options: {
+        requestService: {
```

Comment:
Changed to a wording like https://github.com/apache/inlong/blob/master/inlong-dashboard/src/pages/ProcessDetail/GroupConfig.tsx#L115, the effect is that when entering keywords, you can adjust the API to search, not just search on the front end